Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -934,6 +940,8 @@ def checkpoint(self):
                     b'info-hash': self.tdef.get_infohash()
                 }
                 self.post_alert('save_resume_data_alert', dict(resume_data=resume_data))
+            else:
+                print("either file does not exist or is not file")
```

Comment:
Wait, why not use `logging` here?

Reply:
hmm...

Reply:
Oh, seems like the print statement PR check is broken.

Reply:
As well as the `External import checker`. They are still on Python 2 and dying.

Reply:
I'll make an issue.

Reply:
#5029